examples/apfl_example/client.py

```diff
@@ -40,11 +41,21 @@ def get_criterion(self, config: Config) -> _Loss:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="FL Client Main")
     parser.add_argument("--dataset_path", action="store", type=str, help="Path to the local dataset")
+    parser.add_argument(
+        "--seed",
+        action="store",
+        type=int,
+        help="Seed for the random number generator",
```

Comment:
Super minor: This help string, and the one below, could be slightly more informative. That is, something like "Seed for the random number generators across python, torch, and numpy"